omaha/protocol.go

```diff
@@ -82,7 +82,7 @@ type AppRequest struct {
 	// coreos update engine extensions
 	AlephVersion string `xml:"alephversion,attr,omitempty"`
 	BootId       string `xml:"bootid,attr,omitempty"`
-	MachineID    string `xml:"machineid,attr,omitempty"`
+	MachineId    string `xml:"machineid,attr,omitempty"`
```

Comment:
Well, I've already been using "Id" and "URL" and my logic is that "Id" is an abbreviation rather than an initialism/acronym. However looks like the Go style guide specifically says use "ID": https://github.com/golang/go/wiki/CodeReviewComments#initialisms

So I can drop this and instead convert all my "Id"s to "ID" if you prefer.